Admin/NewsAdmin.php

```diff
@@ -25,4 +25,9 @@ public function __construct($title)
 
         $this->setNavigation(new Navigation($rootNavigationItem));
     }
+
+    public function getJsBundleName()
+    {
+        return 'examplenews';
```

Comment:
this is the lowercase bundle short name - following symfony2 standard http://symfony.com/doc/current/cookbook/bundles/best_practices.html